app/src/main/java/nextstep/jwp/model/http/httpresponse/HttpResponse.java

```diff
@@ -0,0 +1,83 @@
+package nextstep.jwp.model.http.httpresponse;
+
+import java.nio.charset.StandardCharsets;
+import nextstep.jwp.model.http.httpheader.HttpHeaderType;
+import nextstep.jwp.model.http.httpheader.HttpHeaders;
+import nextstep.jwp.service.FileService;
+
+public class HttpResponse {
+
+    private HttpStatusLine httpStatusLine;
+    private HttpHeaders httpHeaders = new HttpHeaders();
+    private byte[] body;
+
+    public HttpResponse(HttpStatusLine httpStatusLine, byte[] body) {
+        this.httpStatusLine = httpStatusLine;
+        this.body = body;
+    }
+
+    public static HttpResponse ok(String type, String httpVersion, byte[] body, String eTag) {
+        HttpResponse httpResponse = new HttpResponse(new HttpStatusLine(httpVersion, 200), body);
```

Comment:
```suggestion
        HttpResponse httpResponse = new HttpResponse(new HttpStatusLine(httpVersion, HttpStatusCode.OK), body);
```

enum을 적극적으로 활용하는 건 어떨까요